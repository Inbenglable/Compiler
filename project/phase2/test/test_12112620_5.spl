struct A{
    int a;
    float a;
    char a;
}a;
int foo(int a){
    return a;
}
float boo(float a){
    return a;
}
char poo(char a){
    return a;
}
int main(){
    foo(a.a);
    boo(a.a);
    poo(a.a);
}