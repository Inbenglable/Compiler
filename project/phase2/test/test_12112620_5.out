Error type 3 at line 3: variable "a" is redefined in the same scope
Error type 3 at line 4: variable "a" is redefined in the same scope
Error type 9 at line 17: invalid argument type
Error type 9 at line 18: invalid argument type
